app/schemas.py

```diff
@@ -198,6 +198,11 @@ class PriceCreate(BaseModel):
         description="price of the product, without its currency, taxes included.",
         examples=["1.99"],
     )
+    price_is_discounted: bool | None = Field(
```

Comment:
`bool` instead of `bool | None` no?

Reply:
I wasn't sure what would happen if the field wasn't provided on create. I'll double check and fix 👍

Reply:
ok removed :ok_hand: